pkg/system/phase2_creating.go

```diff
@@ -511,6 +511,16 @@ func (r *Reconciler) SetDesiredCoreApp() error {
 				}}
 				util.MergeVolumeMountList(&c.VolumeMounts, &configMapVolumeMounts)
 			}
+
+			// We are mounting the log path inside 'noobaa-log-processor' container here
+			// TODO: need to update the logic and actual params here, currently hardcoded
+			if false {
```

Comment:
maybe check if we use bucket logging in noobaa spec... it will mean for sure that we will need to mount something to the pods, if a pvc that we are creating or one provided.

Reply:
sure.